pkg/discovery/dtofactory/application_entity_dto_builder.go

```diff
@@ -98,9 +98,9 @@ func (builder *applicationEntityDTOBuilder) BuildEntityDTO(pod *api.Pod) ([]*pro
 		appType := util.GetAppType(pod)
 		entityDTO, err := ebuilder.
 			ApplicationData(&proto.EntityDTO_ApplicationData{
-				Type:                    &appType,
-				IpAddress:               &(pod.Status.PodIP),
-				HostingNodeCpuFrequency: &nodeCPUFrequency,
+				Type:      &appType,
+				IpAddress: &(pod.Status.PodIP),
+				//HostingNodeCpuFrequency: &nodeCPUFrequency,
```

Comment:
Is there any concern of setting `HostingNodeCpuFrequency` on Applicationdata so that you commented out? We need this field for some logic of CPU millicore support on the server side.

Reply:
I have the same question.